### ISSUE TITLE
Attachment pane does not show up in French language in v1.1.0

### ISSUE THREAD

**Issue by larmarange** [nextcloud/deck#2343]
**Describe the bug**

Since update to 1.1.0, the attachment pan disappear in the sidepan.

![image](https://user-images.githubusercontent.com/966307/94999438-b80ed300-05b9-11eb-9b80-32e51779cb74.png)

**Comment by stefan-niedermann** [comment]
Please fill in the complete issue template, it is there for a reason :wink:

Is there an error in the browser console? (Press <kbd>F12</kbd>)

**Comment by markuman** [comment]
I cannot reproduce (Nextcloud 19.0.3, Deck 1.1.0)

![Screenshot_2020-10-04 Deck - Nextcloud(2)](https://user-images.githubusercontent.com/3920157/95010704-c72e6900-062b-11eb-82e4-97bca89e5d17.png)

**Comment by larmarange** [comment]
**Client details:**
 - OS: Windows
 - Browser: tested with Chome, Firefox and Edge.
 - Device: PC

<details>
<summary>Server details</summary>

Web server provided by Ionos (mutualised PHP server)

**Operating system**: Linux (do not know which version) 

**Web server:** Apache

**Database:** MySQL

**PHP version:** 7.3

**Nextcloud version:** 19.0.3

**Where did you install Nextcloud from:** nextcloud.com

**Signing status:**

```
L'en-tête HTTP "Strict-Transport-Security" n'est pas configurée à au moins "15552000" secondes. Pour renforcer la sécurité, nous recommandons d'activer HSTS comme décrit dans nos conseils de sécurisation ↗.
Aucun cache mémoire n'est configuré. Si possible, configurez un "memcache" pour améliorer les performances. Pour plus d'informations consultez la documentation.
Le module PHP OPcache n'est pas disponible. Pour de meilleures performances, il est recommandé de l'activer dans votre installation de PHP.
Cette instance ne dispose pas de plusieurs modules PHP recommandés. Il est recommandé de les installer pour améliorer les performances, et la compatibilité.
imagick
```

**List of activated apps:**

```
occ $ app:list
Enabled:  
- accessibility: 1.5.0  
- activity: 2.12.0  
- bruteforcesettings: 2.0.1  
- calendar: 2.0.4  
- cloud_federation_api: 1.2.0  
- comments: 1.9.0  
- contactsinteraction: 1.0.0  
- dav: 1.15.0  
- deck: 1.1.0  
- documentserver_community: 0.1.7  
- event_update_notification: 1.0.2  
- external: 3.6.0  
- federatedfilesharing: 1.9.0  
- federation: 1.9.0  
- files: 1.14.0  
- files_external: 1.10.0  
- files_markdown: 2.3.1  
- files_mindmap: 0.0.22  
- files_pdfviewer: 1.8.0  
- files_rightclick: 0.16.0  
- files_sharing: 1.11.0  
- files_trashbin: 1.9.0  
- files_versions: 1.12.0  
- files_videoplayer: 1.8.0  
- groupfolders: 7.0.0  
- logreader: 2.4.0  
- lookup_server_connector: 1.7.0  
- nextcloud_announcements: 1.8.0  
- notifications: 2.7.0  
- oauth2: 1.7.0  
- occweb: 0.0.7  
- onlyoffice: 6.0.2  
- password_policy: 1.9.1  
- photos: 1.1.0  
- privacy: 1.3.0  
- provisioning_api: 1.9.0  
- recommendations: 0.7.0  
- serverinfo: 1.9.0  
- settings: 1.1.0  
- sharebymail: 1.9.0  
- spreed: 9.0.4  
- support: 1.2.1  
- survey_client: 1.7.0  
- text: 3.0.1  
- theming: 1.10.0  
- twofactor_backupcodes: 1.8.0  
- updatenotification: 1.9.0  
- viewer: 1.3.0  
- workflowengine: 2.1.0

Disabled:  - admin_audit  - drawio  - encryption  - files_lock  - firstrunwizard  - hsts  - systemtags  - user_ldap
```

**Nextcloud configuration:**

```
  - user_ldap
occ $ config:list system
{    "system": {        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.okyakusama.fr"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "19.0.3.1",
        "overwrite.cli.url": "http:\/\/cloud.okyakusama.fr",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "filelocking.enabled": true,
        "mail_smtpmode": "sendmail",
        "mail_sendmailmode": "pipe",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "theme": "",
        "loglevel": 2,
        "default_language": "fr",
        "default_locale": "fr_FR",
        "versions_retention_obligation": "disabled",
        "app.mail.transport": "php-mail",
        "updater.release.channel": "stable",
        "app_install_overwrite": [            "occweb"        ],
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }}
```

**Are you using an external user-backend, if yes which one:** No

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
This file is empty
```

#### Browser log
```
JQMIGRATE: Migrate is installed, version 1.4.1 jquery-migrate.min.js:2:551
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 2 globals.js:61:15
Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:61:15
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 3 globals.js:61:15
Notifications permissions not yet requested App.vue:301
jQuery is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
New search handler registered search.js:67:12
session heartbeat polling started session-heartbeat.js:97:9
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 5 globals.js:61:15
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 34 globals.js:61:15
Could not find capabilities initial state fall back to _oc_capabilities index.js:14:12
Could not find capabilities initial state fall back to _oc_capabilities 4 index.js:14:12
Ignoring invalid tab 
Object { tag: "vue-component-32-AppSidebarTab", data: {…}, children: undefined, text: undefined, elm: section#tab-attachments, ns: undefined, context: {…}, fnContext: undefined, fnOptions: undefined, fnScopeId: undefined, … }
ncvuecomponents.js:406:41799
Ignoring invalid tab 
Object { tag: "vue-component-32-AppSidebarTab", data: {…}, children: undefined, text: undefined, elm: section#tab-attachments, ns: undefined, context: {…}, fnContext: undefined, fnOptions: undefined, fnScopeId: undefined, … }
ncvuecomponents.js:406:41799
Could not find capabilities initial state fall back to _oc_capabilities index.js:14:12
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
jQuery is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 204 globals.js:61:15
OC.requestToken changed D3Ro17E9wKBxj/PBBwA2faodjESHO5YeASMm558i+fI=:QzAArokWk+Uc4YbqX1NdN913uTKsD6ZSME5gjaZNg58= index.js:312:9
OC.requestToken changed 80FVQvwuYfVP9LbDhry6AIABnmZO8nbsKw0S31Y3sAU=:vwU9O8QFMrAimsPo3u/RSvdrqxBlxkagGmBUtW9Yymg= index.js:312:9

```

</details>

**Comment by larmarange** [comment]
For information, I have also tested Deck on another instance of Nextcloud, hosted by https://cloudeezy.com/, a dedicated provider for Nextcloud.

I do not have all the details about the server configuration. 

Nextcloud: 19.0.3  / Deck 1.1.0

Sometimes, Deck is not loading at all


<details>
<summary>Browser Logs</summary>

#### Browser log
```
JQMIGRATE: Migrate is installed, version 1.4.1 jquery-migrate.min.js:2:551
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:61:15
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 3 globals.js:61:15
Notifications permissions not yet requested App.vue:301
jQuery is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
Uncaught Error: Could not find initial state config of deck
    loadState index.js:19
    <anonymous> main.js:60
    Webpack 8
index.js:19:10
session heartbeat polling started session-heartbeat.js:97:9
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 3 globals.js:61:15
jQuery is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 12 globals.js:61:15
jQuery is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
jQuery is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
jQuery is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
jQuery is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
jQuery is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 84 globals.js:61:15

```

</details>


When Deck is loading, the attachment pane is not visible

<details>
<summary>Browser Logs</summary>

#### Browser log
```
JQMIGRATE: Migrate is installed, version 1.4.1 jquery-migrate.min.js:2:551
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:61:15
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 3 globals.js:61:15
Notifications permissions not yet requested App.vue:301
jQuery is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
New search handler registered search.js:67:12
session heartbeat polling started session-heartbeat.js:97:9
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 3 globals.js:61:15
TypeError: e.data.ocs is undefined
    value OverviewApi.js:37
Overview.vue:148
Could not find capabilities initial state fall back to _oc_capabilities index.js:14:12
Could not find capabilities initial state fall back to _oc_capabilities 3 index.js:14:12
Ignoring invalid tab 
Object { tag: "vue-component-42-AppSidebarTab", data: {…}, children: undefined, text: undefined, elm: section#tab-attachments, ns: undefined, context: {…}, fnContext: undefined, fnOptions: undefined, fnScopeId: undefined, … }
ncvuecomponents.js:406:41799
Ignoring invalid tab 
Object { tag: "vue-component-42-AppSidebarTab", data: {…}, children: undefined, text: undefined, elm: section#tab-attachments, ns: undefined, context: {…}, fnContext: undefined, fnOptions: undefined, fnScopeId: undefined, … }
ncvuecomponents.js:406:41799

```

</details>

**Comment by acsfer** [comment]
Issue comes from the language. In French, "Pièces Jointes" is somehow trowing an error.
Switch to English, error is gone.

@larmarange

**Comment by larmarange** [comment]
Thank you very much for your feedback. I confirm that after language change, the attachment pane is visible.

**Comment by larmarange** [comment]
One additional thing. I'm not sure that this is related to the same issue.

Now in English, I can see the attached files of cards. For new cards, no problem to attach a file and then to download it.

For files who has been attached with previous version of Deck, they are visible in the attachment pan, but it is impossible to download them.

When checking by FTP the folder `appdata_ocbma80hkq5e/deck`, there are sub-folders for each card created before the update, but they are empty.

**Comment by juliushaertl** [comment]
Thanks for the details and for already figuring out how to trigger this, I'll look into it.

**Comment by juliushaertl** [comment]
Upstream fix in the vue components library https://github.com/nextcloud/nextcloud-vue/pull/1431

**Comment by juliushaertl** [comment]
Fixed with `@nextcloud/vue@v2.6.9
